src/platforms/javascript/index.mdx

```diff
@@ -23,119 +23,57 @@ Using a framework? Take a look at our specific guides to get started.
-
-You may also need to manage your configuration if you are using a third-party library to implement promises. Also, remember that browsers often implement security measures that can block error reporting when serving script files from different origins.
-
-Learn more about how to manually capture errors or enable message capture with Sentry's JavaScript SDK in [Advanced Usage](/platforms/javascript/configuration/capture).
+<Alert level="note" title="Note"><markdown>
 
-### Automatically Enrich Error Data
+If you are using our CDN, please see our documentation for [installing with CDN](/platforms/javascript/configuration/install-cdn).
 
-Events sent by the JavaScript SDK to Sentry are enriched with data that helps identify the source of the event. Much of this data is sent automatically - including the error context and environment - as well as the trail of events that led up to the event, which we call breadcrumbs. You don't need to configure these, though you may modify them.
+</markdown></Alert>
 
-Learn more about the data sent to with events in [Event Context](/platforms/javascript/enriching-error-data/additional-data/).
+## Initialize
 
-### Set the Release Version
+`init` Sentry as soon as possible in your app. Once this is done, all unhandled exceptions are automatically captured by Sentry.
 
-When you configure Sentry to include the version of your application, Sentry can tell you about regressions as well as detailed information about the suspect commit. You will also need releases for source maps.
+This snippet includes:
+- the [release](/product/releases/) of your application, using `process.env.npm_package_version`, which both enables [source maps](/platforms/javascript/sourcemaps/) as well as tracking of regressions and suspect commits.
+- our [performance monitoring](/product/performance/) package.
 
-Use the `process.env.npm_package_version`:
+<PlatformContent includePath="initialize-configure" />
 
-```JavaScript
-Sentry.init({
-  dsn: '___PUBLIC_DSN___',
-  release: 'my-project-name@' + process.env.npm_package_version,
-});
-```
+<Alert level="note" title="Note"><markdown>
 
-Learn more about what releases can do in [our documentation for releases](/product/releases/).
+We'll automatically assign you a Data Source Name (DSN), which looks like a standard URL. It's required by Sentry and configures the protocol, public key, server address, and project identifier. If you forget it, view _Settings -> Projects -> Client Keys (DSN)_ in sentry.io.
 
-### Upload Source Maps
+</markdown></Alert>
 
-We **highly recommend** you incorporate source maps to receive the full benefit of error tracking and monitoring. See our [source maps documentation](/platforms/javascript/sourcemaps/) to learn more.
+# Test
```

Comment:
```suggestion
## Test
```

Or was this on purpose a H1?

Reply:
Nope - an error. Thanks for the careful eye.